library/Businessprocess/Web/Component/Dashboard.php

```diff
@@ -88,9 +90,16 @@ protected function __construct(Auth $auth, Storage $storage)
             $meta = $storage->loadMetadata($name);
             $title = $meta->get('Title');
             if ($title) {
-                $title = sprintf('%s (%s)', $title, $name);
+                $title = FormattedString::create(
+                    '%s %s',
+                    HtmlElement::create('span', ['class' => 'text', 'title' => $title], $title),
+                    HtmlElement::create('span', ['class' => 'header-id', 'title' => $name], '(' . $name . ')')
+                );
             } else {
-                $title = $name;
+                $title = FormattedString::create(
+                    '%s',
+                    HtmlElement::create('span', ['class' => 'text', 'title' => $name], $name)
+                );
             }
```

Comment:
Please rebase `master` first.
- The id `($name)` is no longer required, change this accordingly.